rosgraph_monitor/include/rosgraph_monitor/monitor.hpp

```diff
@@ -152,6 +156,7 @@ class RosGraphMonitor
   {
```

Comment:
we probably oughtta add the name into this tracking struct, that way it can serialize itself `to_msg`. Not necessary for this particular diff though, just thinking about it